### PR TITLE
added Netflix data Gibbs sampling in notebook; doc cleaning

### PR DESCRIPTION
**Pull request by inigobenavides**
Applied BPMF Gibbs to sampled Netflix data in notebook; works by mapping sampled users and movies to keys from `1:n_users` and `1:n_movies`, respectively. Can use to reverse map to original `userId` and `movieId`:

image

### CODE REVIEW

File: stat_280_project_netflix_recsys.Rmd [hunk 1]
```diff
@@ -79,8 +74,8 @@ movies %>% head
 # Top 100 Movies by mean ratings with at least 1000 
 ratings %>%
   group_by(movieId) %>%
-  summarise(mean_rating=mean(rating), count=n()) %>% 
-  filter(count > 1000) %>% 
+  summarise(mean_rating=mean(rating), count=n()) %>%
+  filter(n >= 1000) %>% 
   arrange(desc(mean_rating)) %>%
```
Comment (nfrimando) [inline stat_280_project_netflix_recsys.Rmd:79]: ratings has an n column?

Comment (inigobenavides) [reply stat_280_project_netflix_recsys.Rmd:79]: Ah sorry will fix!

Comment (inigobenavides) [reply stat_280_project_netflix_recsys.Rmd:79]: Should be fixed in latest commit

File: stat_280_project_netflix_recsys.Rmd [hunk 2]
```diff
@@ -352,5 +347,91 @@ simulations.dt %>%
...
+# Define initial parameters
+k_estimate <- 10
+alpha <- 1
+mu_0 <- rep(0, k_estimate)
+beta_0 <- 1
+nu_0 <- 1
+W_0 <- diag(k_estimate)
+
+sampled_ratings_matrix <- sampled_ratings_mapped %>%
+  mutate(rating=(rating - 1)/5) %>%
```
Comment (nfrimando) [inline stat_280_project_netflix_recsys.Rmd]: Let's use the mutate(ratings = sapply(ratings, transform_rating_to_score)) function

File: stat_280_project_netflix_recsys.Rmd [hunk 3]
```diff
@@ -352,5 +347,91 @@ simulations.dt %>%
...
+)
+```
+
+```{r}
+sampled_ratings_matrix
+```
+
+
+```{r}
+netflix_bpmf_gibbs_results$Rs[[800]]
```
Comment (nfrimando) [inline stat_280_project_netflix_recsys.Rmd]: might need to transform back to ratings

**Review by nfrimando — CHANGES_REQUESTED** [review]

File: stat_280_project_netflix_recsys.Rmd (hunk 2 above)
Comment (inigobenavides) [reply stat_280_project_netflix_recsys.Rmd]: image
Doesn't seem correct

Comment (inigobenavides) [reply stat_280_project_netflix_recsys.Rmd]: image
Looks good

**Review by nfrimando — APPROVED** [review]

**Pull request merged**